app/scripts/modules/core/src/pipeline/config/stages/bakeManifest/BakeManifestStageForm.tsx

```diff
@@ -0,0 +1,236 @@
+import * as React from 'react';
+import { Option } from 'react-select';
+
+import { StageArtifactSelectorDelegate } from 'core/artifact';
+import { IArtifact, IExpectedArtifact, IPipeline } from 'core/domain';
+import { MapEditor } from 'core/forms';
+import { IFormikStageConfigInjectedProps, IStageConfigProps, StageConfigField } from 'core/pipeline';
+import { CheckboxInput, ReactSelectInput, TextInput } from 'core/presentation';
+
+interface IBakeManifestStageFormProps {
+  updatePipeline: (pipeline: IPipeline) => void;
+}
+
+export class BakeManifestStageForm extends React.Component<
+  IBakeManifestStageFormProps & IFormikStageConfigInjectedProps
+> {
+  public readonly templateRenderers = ['HELM2'];
+
+  public constructor(props: IStageConfigProps & IFormikStageConfigInjectedProps) {
```

Comment:
can probably get rid of this constructor

Reply:
👍